### PR TITLE
add amazon root certificate for the new site, for next

### PR DESCRIPTION
**Pull request by ArneBab**
This is required to make plugin-updating over HTTPS work with the new site,
and we need that as contingency in case connectivity plugins break.

### CODE REVIEW

File: src/freenet/pluginmanager/PluginDownLoaderOfficialHTTPS.java [hunk 1]
```diff
@@ -28,7 +28,13 @@
 
 public class PluginDownLoaderOfficialHTTPS extends PluginDownLoaderURL {
 	private static final String certurlNew = "freenet/clients/http/staticfiles/globalsign.pem";
-	private static final String[] certURLs = new String[] { certurlNew };
+	// amazon root certificate, so we can rekey the S3-based site more easily
+	private static final String certurlAmazon1 = "freenet/clients/http/staticfiles/AmazonRootCA1.pem";
+	private static final String certurlAmazon2 = "freenet/clients/http/staticfiles/AmazonRootCA1.pem";
```
Comment (nextgens) [inline src/freenet/pluginmanager/PluginDownLoaderOfficialHTTPS.java]: Four copies of the same cert isn't what we want...

**Comment by nextgens** [comment]
The URLs need to change (and while you're at it, you might want to upgrade/get rid of the sha1 stuff)

**Comment by nextgens** [comment]
Oh, and there's also Node.fixCertsFiles() that needs updating

**Comment by ArneBab** [comment]
fixCertsFiles() seems to use what’s in the official plugin downloader

**Comment by ArneBab** [comment]
Closing because it’s out of date.

**Pull request closed without merging**